parsers/EIA.py

```diff
@@ -61,6 +130,10 @@ def _fetch_production_or_consumption(zone_key, series_id, session=None,
         raw_data = series.last(24)
```

Comment:
Quoting from https://www.eia.gov/opendata/commands.php

> Dates are formatted as yyyy, yyyyQq, yyyymm, yyyymmdd for annual, quarterly, monthly, and daily/weekly data respectively.

Since the EM uses `YYYYMMDD` format for target_datetimes we should be ok here.

Reply:
agreed! this is (hopefully) unchanged from before.